frontend/packages/ceph-storage-plugin/src/utils/dashboard.ts

```diff
@@ -1,11 +1,11 @@
 import * as _ from 'lodash';
 import { PrometheusResponse } from '@console/internal/components/graphs';
 
-export const getResiliencyProgress = (results: PrometheusResponse): number => {
+export const getResiliencyProgress = (results: PrometheusResponse): string => {
```

Comment:
```suggestion
export const getResiliencyProgress = (results: PrometheusResponse): string | null => {
```

Reply:
Its implicit as I mentioned hence I did not considered  adding.
Also stricnullchecks is not enabled for console.
I can add anyways.